sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/rust/RustFileWriter.java

```diff
@@ -157,6 +156,7 @@ public void postorder(DBSPTypeSemigroup type) {
                         aggregates::*,
                         variant::*,
                     };
+                    #[cfg(test)]
```

Comment:
I see lines like this in the generated code:

```rust
sltsqlvalue::to_sql_row_impl! {
    Tup16<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15>,
    Tup18<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15, T16, T17>,
    Tup11<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10>,
    Tup12<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11>,
    Tup13<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12>,
    Tup14<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13>,
    Tup15<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14>,
}
```

These are not gated by cfg(test)

Reply:
these should be gone now

Reply:
Fixes #2560